testdata/p4_16_errors_outputs/struct_e.p4-stderr

```diff
@@ -4,9 +4,7 @@ struct_e.p4(24): [--Werror=type-error] error: Field data is not a member of stru
 struct_e.p4(16)
 struct s {}
        ^
-struct_e.p4(25): [--Werror=type-error] error: Cannot extract field data from b which has type bit<1>
-        b.data = 5; // no such field
-          ^^^^
-struct_e.p4(25)
-        b.data = 5; // no such field
-        ^
+terminate called after throwing an instance of 'Util::CompilerBug'
+  what():  In file: /home/hemant/int/p4c/lib/crash.cpp:229
+[31mCompiler Bug[0m: Exiting with SIGSEGV
+
```

Comment:
Not sure whether it is a good idea to check in code with known crashes like this?  Not really for me to judge -- I was just skimming the changes and noticed this.

Reply:
I will look into this one in the parent branch and well is this PR repo and get back - thanks.

Reply:
I had added one line of new code (an ::error message) to typeChecker.cpp to debug my code.  I forgot to remove this line of code.  Once I removed it, the test passed.  I have checked in code changes.  thanks.